CODEOWNERS

```diff
@@ -12,4 +12,4 @@
 #  https://help.github.com/en/articles/about-code-owners
 #
 
-*  @bhs @bogdandrutu @sarahnovotny @SergeyKanzhelev @yurishkuro
+*  @bhs @bogdandrutu @sarahnovotny @SergeyKanzhelev @yurishkuro @lizthegrey @ccaraman @mtwo @tedsuo
```

Comment:
Please split the PR into more focused PRs. Currently this PR changes owners as well as things in readme and membership requirements. I do think all are in general ok but we should follow best practices and have smaller more focused PRs. Thank you.

Reply:
I don't think it changes OWNERS, that's a rebase artifact. I'll rebase it again.

Reply:
Looks way better now, thanks